### PR TITLE
smsxml to use wsgi + other improvements

### PR DESCRIPTION
**Pull request by mikahama**

### CODE REVIEW

File: servers/sanat/roles/hhvm/tasks/main.yml [hunk 1]
```diff
@@ -10,6 +10,9 @@
     repo_gpgcheck: no
     enabled: yes
 
+- name: Enable Epel repository
```
Comment (Nikerabbit) [inline servers/sanat/roles/hhvm/tasks/main.yml]: As far as I know HHVM does not need this. Should probably be in another role?

File: servers/sanat/roles/mediawiki/tasks/main.yml [hunk 2]
```diff
@@ -31,6 +31,15 @@
 - name: Install LocalSettings.php
   template: src=LocalSettings.php.j2 dest=/srv/mediawiki/workdir/LocalSettings.php
 
+- name: Copy Composer installer
```
Comment (Nikerabbit) [inline servers/sanat/roles/mediawiki/tasks/main.yml]: Composer is already installed by the `geerlingguy.composer` role in `site.yml`. Why is this required?

File: servers/sanat/roles/hhvm/tasks/main.yml (hunk 1 above)
Comment (mikahama) [inline servers/sanat/roles/hhvm/tasks/main.yml]: I started a fresh Centos virtual machine, and it was unable to install HHVM without epel

File: servers/sanat/roles/mediawiki/tasks/main.yml (hunk 2 above)
Comment (mikahama) [inline servers/sanat/roles/mediawiki/tasks/main.yml]: Maybe the execution order of the scripts should be changed then.

Comment (Nikerabbit) [inline servers/sanat/roles/mediawiki/tasks/main.yml]: Yeah, that role should be before mediawiki.

File: servers/sanat/roles/hhvm/tasks/main.yml (hunk 1 above)
Comment (Nikerabbit) [inline servers/sanat/roles/hhvm/tasks/main.yml]: Okay, that's weird but I cannot test easily now.
